docs/buildRelease.md

```diff
@@ -0,0 +1,53 @@
+# Auro build systems
+The goal of this document is to roughly outline every step of the build + release process in any given Auro component.
+
+## Local development
+When developing and/or working on an existing Auro component, a developer will initially -
+
+1. git clone a repository
+1. Run `nvm use 20` && `npm install` (nvm step optional)
+    1. As a post-install hook, git-hook tool husky installs scripts to the .git directory. More on this in the commit section!
+1. Checkout a new feature branch off main (some branches still use master)
+1. Run `npm run build` - this is a macro to build sass files and generate TypeScript declarations
+    1. This is inconsistent? Some have `build:version` and some do not
+1. Run `npm run build:docs` to bootstrap markdown files from their static HTML
+1. Run `npm run dev` to start their server and watch their files for changes
+
+**When done working on a feature, or otherwise running git commit:**
+
+1. A commit message linter will run (commit lint)
+    1. A precommit hook runs preCommit test linters postinstall
```

Comment:
I am not convinced that all of these are needed as part of commit lint work flow. 

`precommit` just logs out the definition of done, which is fine.

The test and linters all get run as part of the PR actions which seem like a better place. 

Any why run postinstall again? That is just the "welcome to auro" message logging.